ui/src/app/base/selectors/user/user.js

```diff
@@ -1,32 +1,28 @@
+import { createSelector } from "@reduxjs/toolkit";
+
 const user = {};
 
 /**
  * Returns all users
  * @param {Object} state - The redux state.
- * @param {Number} batch - Number of users to return.
```

Comment:
This is the only select where there any api changes. We're not using this batch param in favour of handling the pagination using the table.